sonic_installer/main.py

```diff
@@ -384,7 +384,7 @@ def migrate_sonic_packages(bootloader, binary_image_version):
             run_command_or_raise(["chroot", new_image_mount, SONIC_PACKAGE_MANAGER, "migrate",
```

Comment:
>SONIC_PACKAGE_MANAGER [](http://example.com/codeflow?start=61&length=21)

Could you explain "Don't capture output from migrate command for better debugging"? Do you mean the output is too verbose?

Reply:
@qiluo-msft ```run_command_or_raise``` was capturing stdout from a command using ```subprocess.PIPE``` and the output was returned to the caller. If the caller does not print it we miss some valuable debug information printed during "migrate" command, e.g. why some extensions have failed to migrate, etc.
Since, we don't need to capture and process the output I decided to add a flag that will tell ```run_command_or_raise``` to not capture stdout and have it printed to the console.